services/brig/src/Brig/API/Public.hs

```diff
@@ -1030,25 +1030,22 @@ getUserDisplayNameH (_ ::: self) = do
     Just n -> json $ object ["name" .= n]
     Nothing -> setStatus status404 empty
 
-listUsersH :: JSON ::: UserId ::: Either (List OpaqueUserId) (List (OptionallyQualified Handle)) -> Handler Response
+listUsersH :: JSON ::: UserId ::: Either (List OpaqueUserId) (Range 1 4 (List (OptionallyQualified Handle))) -> Handler Response
```

Comment:
Motivation for this change is that the check for handles is rather expensive - and clients do not use this. Checking with the different client teams I got the information that only 1 handle is ever looked up.

Reply:
b7f9e02c88ec81066c4d5af1f629b6eccc00f723